src/rtcm.h

```diff
@@ -60,6 +60,7 @@ class RTCMParsing
 
 	uint8_t *message() const { return _buffer; }
 	uint16_t messageLength() const { return _pos; }
+	uint16_t messageId() const { return (_buffer[3]<<4)|(_buffer[4]>>4); }
```

Comment:
Style check will probably complain in PX4:
```suggestion
	uint16_t messageId() const { return (_buffer[3] << 4) | (_buffer[4] >> 4); }
```